packages/components/input-number/src/input-number.vue

```diff
@@ -278,6 +278,9 @@ watch(
     if (!isNumber(userInput) && (!userInput || userInput !== newValue)) {
       data.currentValue = newValue
       data.userInput = null
+    } else if (isNumber(userInput) && userInput !== data.currentValue) {
+      data.currentValue = userInput
```

Comment:
This change will invalidate the decimal point input

Reply:
If it does, then there's a unit test missing for your comment. When I created that PR all tests had passed.